src/app/shared/menu/date/standard-date-range.component.ts

```diff
@@ -1,47 +1,52 @@
 import { Component, Input, Output, EventEmitter, OnChanges } from '@angular/core';
-import { IntervalModel, INTERVAL_HOURLY, INTERVAL_DAILY, INTERVAL_WEEKLY, INTERVAL_MONTHLY } from '../../../ngrx';
-import { isMoreThanXDays, endOfTodayUTC,
-  beginningOfThreeMonthsUTC, beginningOfThisYearUTC,
-  TODAY, THIS_WEEK, TWO_WEEKS, THIS_MONTH, THREE_MONTHS, THIS_YEAR,
-  YESTERDAY, LAST_WEEK, PRIOR_TWO_WEEKS, LAST_MONTH, PRIOR_THREE_MONTHS, LAST_YEAR } from '../../util/date.util';
+import { IntervalModel, INTERVAL_HOURLY, INTERVAL_MONTHLY } from '../../../ngrx';
+import { dayOfWeekDateFormat, dayMonthDateFormat, monthDateYearFormat,
```

Comment:
So many imports!  Wondering if this is a case where `import * as dateUtils from 'date.util'` would actually clean this up.